### PR TITLE
{CI} Pipeline to auto sync resourceManagement.yml according to ADO Wiki Page - Service Contact List

### PR DESCRIPTION
**Pull request by Pan-Qi**
This PR creates a pipeline that automatically syncs the alias in the `resourceManagement.yml` file according to the ADO Wiki Page - Service Contact List on a scheduled basis.

trigger on feature branch is only for testing:
```
trigger:
  - bernard-auto-update-service-owner
```
And will need to be replaced with schedules before merging:
```
#schedules:
#- cron: "50 15 * * *"
#  displayName: 11:50 PM (UTC + 8:00) China Daily Run
#  branches:
#    include:
#    - dev
```

This [PR ](https://github.com/Azure/azure-cli/pull/30027) was automatically generated by the sync-alias pipeline. It is based on the current feature branch, which includes code related to the sync-alias pipeline.
However, after this PR is merged, future automatically scheduled PRs will only contain changes related to the .github/policies/resourceManagement.yml file.

**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: .azure-pipelines/sync-alias.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+#schedules:
+#- cron: "50 15 * * *"
+#  displayName: 11:50 PM (UTC + 8:00) China Daily Run
+#  branches:
+#    include:
+#    - main
+
+trigger:
+  - bernard-auto-update-service-owner
```
Comment (wangzelin007) [inline .azure-pipelines/sync-alias.yml:11]: Please rename to `auto-update-service-owner`.

**Review by wangzelin007 — DISMISSED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Comment by yonzhan** [comment]
Please fix CI issues

**Pull request merged**